src/App.css

```diff
@@ -1,56 +1,261 @@
 * {
   list-style-type: none;
+  text-decoration: none;
+  box-sizing: border-box;
 }
 
-form {
+/* navbar */
+header {
+  background-color: white;
+  box-shadow: rgba(0, 0, 0, 0.05) 0 4px 6px -2px;
+}
+
+nav {
+  height: 80px;
   display: flex;
-  gap: 10px;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 10rem;
+  font-weight: bolder;
 }
 
-button {
-  margin-left: 30px;
-  padding: 5px 10px;
-  background-color: rgb(0, 183, 255);
-  color: azure;
-  border: none;
+.left {
+  display: flex;
+  justify-content: space-between;
 }
 
-input {
-  padding: 5px 10px;
+.title {
+  color: rgba(41, 113, 208, 0.723);
 }
 
-nav {
+.menu {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  margin-block: 10px;
-  padding: 10px 15px;
+  gap: 20px;
+  line-height: 47px;
+  font-size: 13px;
 }
 
+.avatar {
+  vertical-align: middle;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+}
+
+/* booklist styles */
 .booklist {
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr;
+  border: 1px solid #e8e8e8;
+  max-width: 1215px;
+  margin: 30px 110px;
+  gap: 5px;
+  border-radius: 5px;
+  padding: 20px;
+  box-shadow: rgba(0, 0, 0, 0.1) 0 10px 15px -3px, rgba(0, 0, 0, 0.05) 0 4px 6px -2px;
+}
+
+.book {
+  font-family: 'Roboto Slab', sans-serif;
+  margin-block: 15px;
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
+  gap: 5px;
+}
```

Comment:
- [ ] Great effort working on the design of the Bookstore project, I would kindly suggest you stick to the original design as much as possible, Currently I notice the `profile icon` and the `add book` button does not align vertically on a straight line with the books added. Kindly check the original design to see how they all align vertically together on a straight line.
- [ ] Also the text in the `Add book` button is on a straight line in the original but what we have in your design is broken on two lines. Kindly work on it.

Your Design 👇 

![image](https://github.com/rabayarabu/bookstore/assets/78634841/3b7edd20-ac6e-43ac-b165-4a2cb1a6af62)